Makefile

```diff
@@ -55,6 +55,21 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+.PHONY: cma-check-all-csv
+cma-check-all-csv: build-testutil ## Verify that CMA CSV files look right
+	hack/cma-check-all-csv.sh
+
+.PHONY: build-testutil
+build-testutil: bin/yaml2json bin/json2yaml ## Build utilities needed by tests
+
+# utilities needed by tests
+bin/yaml2json: cmd/testutil/yaml2json/yaml2json.go
+	mkdir -p bin
+	go build $(GOGCFLAGS) -ldflags "$(LD_FLAGS)" -o bin/ "github.com/kedacore/keda-olm-operator/cmd/testutil/yaml2json"
```

Comment:
Is this a valid path?  `github.com/openshift/custom-metrics-autoscaler-operator/cmd/testutil/yaml2json` maybe?

Reply:
That's a good question... The way I have my repo set up is that I cloned `github.com/kedacore/keda-olm-operator` then added a new remote of this repo. I think that as long as the CI does it the same way, it should be valid.

Reply:
Okay, once the following PR merges, I think the path should be valid. I changed the `canonical_go_repository` to be `github.com/kedacore/keda-olm-operator` so that the CI source should be set up in a location like mine.

https://github.com/openshift/release/pull/36105